src/Entity/Activity.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ActivityRepository")
+ */
+class Activity
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(type="string", length=30)
```

Comment:
Usually the names are longer. The default length of `string` is 255. Please remove the length attribute. The change should be reflected in the other classes also.